extensions/amp-analytics/0.1/vendors.js

```diff
@@ -297,5 +297,40 @@ export const ANALYTICS_CONFIG = {
       'pageview': 'https://${host}${requestPath}?${basePrefix}',
       'click': 'https://${host}${requestPath}?${basePrefix}&pe=lnk_${linkType}&pev1=${linkUrl}&pev2=${linkName}',
     }
+  },
+
+  'webtrekk': {
+    'requests': {
+      'trackURL': 'https://${trackDomain}/${trackId}/wt',
+      'parameterPrefix': '?p=431,${contentId},1,' +
+          '${screenWidth}x${screenHeight},${screenColorDepth},' +
+          '${documentReferrer},${timestamp},0,,0&tz=${timezone}' +
+          '&eid=${clientId(amp-wt3-eid)}&la=${browserLanguage}',
+      'parameterSuffix': '&pu=${canonicalUrl}',
+      'pageParameter': '&cp1=${pageParameter1}' +
+          '&cp2=${pageParameter2}&cp3=${pageParameter3}' +
```

Comment:
I am assuming that these can be sometimes specified by the publishers. If that is the case,  you can probably get a cleaner config using https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/amp-analytics.md#extra-url-params 

In that case, you just need to specify a mapping from pageParameter to cp,  pageCategory to cg and actionParameter to ck. Then your customers just need to specify the parameters in extraUrlParams section and they will get included in the requests. 

If this works for you, please make the change. If not, just let us know.